types/sse4_crc32/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for sse4_crc32 7.0
+// Project: https://github.com/anandsuresh/sse4_crc32#readme
+// Definitions by: naeemy <https://github.com/naeemy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { ReadStream } from 'fs';
+import { Writable } from 'stream';
+
+export class Crc32CStream extends Writable {
+    constructor(crc?: number);
+}
+
+/**
+ *
+ * @param stream Readable stream object
+ * @param crc Non-zero integer value
+ */
+export function fromStream(stream: ReadStream, crc?: number): Crc32CStream;
+/**
+ *
+ * @param buf Buffer object or string
+ * @param initial Non-zero integer value
+ */
+export function calculate(buf: Buffer | string, initial?: number): number;
+
+// only available when NODE_ENV='production'
+export const hardware_support: boolean;
```

Comment:
what about?
```suggestion
export const hardware_support?: boolean | undefined;
```
so people can enforce this explicitly when debugging/benchmarking
```ts
mod.hardware_support!;
```